accepted/2022/dotnet-nuget-why-proposal.md

```diff
@@ -118,6 +118,11 @@ dotnet nuget why [<PROJECT>|<SOLUTION>] <PACKAGE_NAME>
 
 dotnet nuget why -h|--help
 ```
+
+We will show the resolved versions of packages in the graph, rather than the versions requested by the top-level package.
+
+We will also deduplicate graphs in certain cases. If we have already shown a package's dependencies in the graph in another place, we will truncate the graph at that node and show a `(*)` symbol to indicate that the package's dependencies have already been shown. Further, if multiple target frameworks in a project have the same dependency graph, we will show a combined graph for them. (See [examples](#examples))
```

Comment:
```suggestion
We will also deduplicate graphs in certain cases.
If we have already shown a package's dependencies in the graph in another place, we will truncate the graph at that node and show a `(*)` symbol to indicate that the package's dependencies have already been shown.
Further, if multiple target frameworks in a project have the same dependency graph, we will show a combined graph for them.
(See [examples](#examples))
```

I don't know if it's widely considered best practice, but I certainly consider it a markdown best practice to put 1 sentence per line. Markdown renders it the same way, but in the future if it gets modified, the diff will be clearer. GitHib in particular has troubles with diffs on very long lines. Plus it allows people commenting in PRs to comment on a single sentence, rather than the entire pargraph.

Reply:
Thanks! I'll start doing that